cogs/mod.py

```diff
@@ -1003,7 +1003,13 @@ def check(m):
     async def blacklist(self, ctx):
         """Bans user from using the bot"""
         if ctx.invoked_subcommand is None:
+            name = list(set([x.name for x in self.bot.get_all_members() if x.id in self.blacklist_list]))
             await send_cmd_help(ctx)
+            if name:
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
+            await self.bot.say(box('Blacklisted users:\n' + userlist))
```

Comment:
Unsure if what to display ontop.
My suggestions are

- Blacklisted users :
- Blacklist :